metricbeat/etc/fields.yml

```diff
@@ -8,7 +8,28 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["general", "General event data"]
   - ["redis-info", "Redis Info Stats"]
+
+general:
+  type: group
+  description: >
+    General fields for each event
+  fields:
+    - name: response_time
```

Comment:
I like `response_time` more than `responsetime`. And I like `response_time_nanos` even more.